libfwupdplugin/fu-udev-device.c

```diff
@@ -518,6 +518,114 @@ fu_udev_device_get_slot_depth (FuUdevDevice *self, const gchar *subsystem)
 	return 0;
 }
 
+#ifndef _WIN32
+static gchar *
+fu_udev_device_get_bind_id (FuUdevDevice *self)
+{
+	FuUdevDevicePrivate *priv = GET_PRIVATE (self);
+	if (g_strcmp0 (fu_udev_device_get_subsystem (self), "pci") == 0)
+		return g_strdup (g_udev_device_get_property (priv->udev_device, "PCI_SLOT_NAME"));
+	if (g_strcmp0 (fu_udev_device_get_subsystem (self), "usb") == 0)
+		return g_path_get_basename (g_udev_device_get_sysfs_path (priv->udev_device));
```

Comment:
for I2C it seems to be the stuff in `PhysicalId`

`HID_PHYS=i2c-CUST0001:00`

Reply:
What's the subsystem tho, `hid`, or `i2c`?

Reply:
This was from the synaptics-rmi device.

```
Touchpad
  DeviceId:             b26933c085b020ecf84c490812458523aee710ac
  Guid:                 f4384034-9243-5334-8075-a534be913e46 <- HIDRAW\VEN_06CB&DEV_76AF&REV_00
  Guid:                 424bd00e-9789-5cdf-a12a-3c81bc4676d6 <- HIDRAW\VEN_06CB&DEV_76AF
  Guid:                 140f4458-951b-5bb9-85e2-879bd5b02615 <- SYNAPTICS_RMI\TM3038-003
  Guid:                 b29d3c85-cd0e-503e-9c7e-f6731c1eaf2d <- SYNAPTICS_RMI\TM3038
  Plugin:               synaptics_rmi
  Protocol:             com.synaptics.rmi
  Flags:                internal|updatable|registered
  Vendor:               Synaptics
  VendorId:             HIDRAW:0x06CB
  Version:              1.5.2767034
  VersionBootloader:    54.0.0
  VersionFormat:        triplet
  Created:              2020-09-21
  PhysicalId:           HID_PHYS=i2c-CUST0001:00
  SysfsPath:            /sys/devices/pci0000:00/0000:00:15.1/i2c_designware.1/i2c-2/i2c-CUST0001:00/0018:06CB:76AF.0001/hidraw/hidraw0
  BlVer:                0x6
  BootloaderId:         56.36
  BlockSize:            0x10
  BlockCountFw:         0xe60
  BlockCountCfg:        0x40
  FlashConfigLength:    0x0
  PayloadLength:        0x0
  BuildID:              0x2a38ba
```

Reply:
You need to use `FU_UDEV_DEVICE_DEBUG=1` to see the subsystem iirc.

Reply:
Oh it's hid - you're right.

Reply:
Okay, added, thanks.